### PR TITLE
New funnie table flip.

### PR DESCRIPTION
**Pull request by 0Moose0**

### CODE REVIEW

File: looks.yml [hunk 1]
```diff
@@ -1099,3 +1099,6 @@
 - plain: "(∩ ` -´)⊃━━☆ﾟ.*･｡ﾟ"
   title: Wizard
   tags: all
+- plain "(☞ﾟヮﾟ)☞ ┻━┻"
```
Comment (leighmcculloch) [inline looks.yml:1102]:
```suggestion
- plain: "(☞ﾟヮﾟ)☞ ┻━┻"
```

File: looks.yml [hunk 2]
```diff
@@ -1099,3 +1099,6 @@
...
+  Title: pointing at flipped table
```
Comment (leighmcculloch) [inline looks.yml:1103]:
```suggestion
  title: Pointing at flipped table
```

File: looks.yml [hunk 3]
```diff
@@ -1099,3 +1099,6 @@
...
+  tags: all flipping tables
```
Comment (leighmcculloch) [inline looks.yml:1104]:
```suggestion
  tags: all flipping-tables
```

**Review by leighmcculloch — CHANGES_REQUESTED** [review]
Thanks for adding a look! A couple small things need fixing then we can merge it.

**Comment by 0Moose0** [comment]
so does that mean its on the site?

**Review by leighmcculloch — APPROVED** [review]

**Comment by leighmcculloch** [comment]
> so does that mean its on the site?

It will take a little bit for the cached pages in the CDN to expire, but it is deployed!

**Comment by 0Moose0** [comment]
Thank you!

**Pull request merged**